files/presets/Nintendo Pokemon Mini.json

```diff
@@ -0,0 +1,81 @@
+{
+    "Nintendo Pokemon Mini - Retroarch": {
+        "parserType": "Glob",
+        "configTitle": "Pokemon Mini - Retroarch",
+        "steamDirectory": "${steamdirglobal}",
+        "steamCategory": "${Pokemon Mini}",
+        "romDirectory": "path-to-roms",
+        "executableArgs": "run org.libretro.RetroArch -L ${os:win|cores|${os:mac|${racores}|${os:linux|${racores}}}}${/}pokemini_libretro.${os:win|dll|${os:mac|dylib|${os:linux|so}}} \"${filePath}\"",
+        "executableModifier": "\"${exePath}\"",
+        "startInDirectory": "",
+        "titleModifier": "${fuzzyTitle}",
+        "fetchControllerTemplatesButton": null,
+        "removeControllersButton": null,
+        "imageProviders": [
+            "SteamGridDB"
+        ],
+        "onlineImageQueries": "${${fuzzyTitle}}",
+        "imagePool": "${fuzzyTitle}",
+        "userAccounts": {
+            "specifiedAccounts": ""
+        },
+        "executable": {
+            "path": "${retroarchpath}",
```

Comment:
`/usr/bin/flatpak`

Executable args show this is a Flatpak preset, provide Flatpak binary prefilled